tests/fastcgi-auth.conf

```diff
@@ -68,15 +68,23 @@ compress.filetype = (
 fastcgi.debug = 0
 fastcgi.server = (
 	"/" => (
-			"grisu" => (
+		"grisu" => (
 			"host" => "127.0.0.1",
 			"port" => 20000,
 			"bin-path" => env.SRCDIR + "/fcgi-auth",
 			"mode" => "authorizer",
-			"docroot" => env.SRCDIR + "/tmp/lighttpd/servers/www.example.org/pages/",
```

Comment:
In short: No.
The configuration is only used for the FastCGI authorizer tests. These were previously 3, and are now 5. The requested files (`/index.html` and `/expire/access.txt`) are properly delivered, because `server.document-root` is the same as `docroot` and the request is now reloaded, instead of delivering directly out of `docroot` (which is also possible, I haven't removed the configuration option).

All the tests pass fine also:
`````
./mod-fastcgi.t ....... ok     
[...]
All tests successful.
Files=25, Tests=403,  6 wallclock secs ( 0.13 usr  0.02 sys +  1.27 cusr  0.17 csys =  1.59 CPU)
Result: PASS
PASS: run-tests.pl
cleaning up                             PASS: cleanup.sh
==================
All 3 tests passed
==================
`````